src/60_Samples_%26_Templates/Live_Blog.html

```diff
@@ -1,8 +1,6 @@
 <!---
 
 preview: default
```

Comment:
my bad, we disabled validation because of the go template literals not passing validation

Reply:
right, I reverted the change